### DISCUSSION TITLE
Not finding dlls

### DISCUSSION THREAD

**Discussion by peranltmp** [cnr-isti-vclab/TagLab#19] [Q&A] [answered]
When I run taglab 0.4 I get errors saying that it can't load various dll files

ERROR 1: Can't load requested DLL: C:\Program Files\GDAL\gdalplugins\ogr_FileGDB.dll
127: The specified procedure could not be found.

taglab still loads but appears to have reduced functionality including things like the fully automatic classification forcing taglab to exit

I installed gdal and its extensions using the instructions found here

https://sandbox.idre.ucla.edu/sandbox/tutorials/installing-gdal-for-windows

any ideas?

**Comment by alemuntoni** [comment, marked as answer]
Hi,
did you run `install.py`? It shouldn't be necessary to install gdal in the way you linked.
